Text/HealthcareTA/HealthcareTA.cs

```diff
@@ -0,0 +1,136 @@
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.WebJobs;
+using Microsoft.Azure.WebJobs.Extensions.Http;
+using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System.Collections.Generic;
+using AzureCognitiveSearch.PowerSkills.Common;
+using Azure.AI.TextAnalytics;
+using Azure;
+
+namespace AzureCognitiveSearch.PowerSkills.Text.HealthcareTA
+{
```

Comment:
Can we call this "TextAnalyticsForHealth" since that is the official name they use? Make sure you update the project and folder name too.